src/voting/templatetags/voting_tags.py

```diff
@@ -64,8 +64,8 @@ def __init__(self, user, objects, context_var):
 
     def render(self, context):
         try:
-            user = template.resolve_variable(self.user, context)
-            objects = template.resolve_variable(self.objects, context)
+            user = template.Variable(self.user).resolve(context)
+            objects = template.Variable(self.object).resolve(context)
```

Comment:
i think i was working with a pre-release version of django1.11 if that's the case ...